### PR TITLE
Uploading assignments 20 21

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

**Review by sanketmod — APPROVED** [review]

File: poonam/JS/Assignment20_js/que2.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            </tr>
+        </table>
+        <br>
+        Enter no of row: <input type="text" id="rowno"><br><br>
+        Enter no of column: <input type="text" id="colno"><br><br>
+        Enter data to update: <input type="text" id="data"><br><br>
+        <button onclick="add()">Add row</button>
+        <button onclick="changeContent()">Update</button>
+        <script>
+            function add() {
```
Comment (MKadeer17) [inline poonam/JS/Assignment20_js/que2.html:42]: This add row Function not working properly.
```suggestion
            function add() {
```

**Review by MKadeer17 — APPROVED** [review]

File: poonam/JS/Assignment20_js/que2.html (hunk 1 above)
Comment (poonam-khode) [reply poonam/JS/Assignment20_js/que2.html:42]: it is working on my side

Comment (MKadeer17) [reply poonam/JS/Assignment20_js/que2.html:42]: But it's added only empty row not value with row.

Comment (poonam-khode) [reply poonam/JS/Assignment20_js/que2.html:42]: i added changeContent function for adding data

**Pull request merged**